samples/openthread/cli/sample.yaml

```diff
@@ -12,10 +12,9 @@ tests:
       - nrf5340dk_nrf5340_cpuapp
   samples.openthread.cli.thread_1_2:
     build_only: true
-    platform_allow: nrf52840dk_nrf52840 nrf52833dk_nrf52833 nrf5340dk_nrf5340_cpuapp
```

Comment:
The  commit message does not mention this change. Is Thread 1.2 not supported by nRF53 (yet)?

Reply:
That's correct. nRF53 does not support all Thread v1.2 features yet.

Reply:
The commit message still doesn't mention why nRF5340dk is being removed.
Current commit message:
>  samples: openthread: enable CSL Receiver for Thread 1.2
>
> Enable CSL Receiver building for Thread 1.2 overlay.

Please update the commit message